cmd/validator/flags/BUILD.bazel

```diff
@@ -8,7 +8,7 @@ go_library(
     ],
     importpath = "github.com/prysmaticlabs/prysm/cmd/validator/flags",
     visibility = [
-        "//cmd/validator:__subpackages__",
+        "//cmd:__subpackages__",
```

Comment:
Why change this?

Reply:
removing this. was trying to do it the first way.